catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrCatalogProvider.java

```diff
@@ -330,7 +331,7 @@ public CreateResponse create(CreateRequest request) throws IngestException {
         } catch (SolrServerException | SolrException | IOException | MetacardCreationException e) {
             throw new IngestException("Solr could not ingest metacard(s).");
         }
-
+        // TODO (RCZ) - Do we need to put the req props into the response????
```

Comment:
its there because its a question. Should we be putting the request props into the response?